isis/src/base/apps/caminfo/caminfo.xml

```diff
@@ -39,164 +39,212 @@
 Object = Caminfo
   Object = Parameters
     Program     = caminfo
-    IsisVersion = "3.7.0  | 2019-04-16"
-    RunDate     = 2019-04-24T23:29:22
-    IsisId      = MeSSEnGeR/MDIS-WAC/1/0211286081:889000
-    From        = EW0211286081G.lev1.cub
-    Lines       = 1024
-    Samples     = 1024
+    IsisVersion = "6.0.0  | 2021-08-10"
+    RunDate     = 2021-08-18T14:53:40
+    IsisId      = Viking1/VISB/33322515
+    From        = default.cub
+    Lines       = 1056
+    Samples     = 1204
     Bands       = 1
   End_Object
 
   Object = Camstats
-    MinimumLatitude           = 46.17349725649
-    MaximumLatitude           = 54.072584907617
-    MinimumLongitude          = 284.17017788557
-    MaximumLongitude          = 296.47498126014
-    MinimumResolution         = 235.10973750118
-    MaximumResolution         = 238.32543228025
-    MinimumPhase              = 50.107674499885
-    MaximumPhase              = 60.748154060087
-    MinimumEmission           = 0.0076793870352429
-    MaximumEmission           = 11.494912087698
-    MinimumIncidence          = 52.486663651971
-    MaximumIncidence          = 58.235714475896
-    LocalTimeMinimum          = 9.7555337547863
-    LocalTimeMaximum          = 10.575853979758
-    ObliqueResolutionMinimum = 235.23505597394
-    ObliqueResolutionMaximum  = 243.18035398447
```

Comment:
These values should not be removed although they are redundant. This would be API breaking and removes preexisting keyword/values that existing pipelines almost assuradly rely on.

You can see in my output I left them in primarily for this reason.